From 649d2b7c0330f57a1532a992f6ce81fcf4751cd4 Mon Sep 17 00:00:00 2001
From: onodera-punpun <onodera@openmailbox.org>
Date: Wed, 28 Oct 2015 21:33:50 +0100
Subject: [PATCH] Make menu spawn under cursor

---
 openbox/menuframe.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/openbox/menuframe.c b/openbox/menuframe.c
index c37fdcc54..929e80c5b 100644
--- a/openbox/menuframe.c
+++ b/openbox/menuframe.c
@@ -269,8 +269,8 @@ static void menu_frame_place_topmenu(ObMenuFrame *self, const GravityPoint *pos,
     } else {
         gint myx, myy;
 
-        myx = *x;
-        myy = *y;
+        myx = *x - 22;
+        myy = *y - 22;
 
         /* try to the bottom right of the cursor */
         menu_frame_move_on_screen(self, myx, myy, &dx, &dy);
